src/components/topmenu.tsx

```diff
@@ -37,7 +37,7 @@ class TopMenu extends PureComponent<TopMenuProps, TopMenuState> {
 								<Dropdown.Item onClick={() => navigate('/bb')}>Text only</Dropdown.Item>
 							</Dropdown.Menu>
 						</Dropdown>
-						<Menu.Item onClick={() => navigate('/voyage')}>Player tools</Menu.Item>
+						<Menu.Item onClick={() => navigate('/playertools')}>Player tools</Menu.Item>
```

Comment:
Can we make a redirect component from /voyage? A bunch of people will have that bookmarked, and I am guessing some external sites link directly there.